src/app/mesh/qgsmeshdatasetgrouptreeview.h

```diff
@@ -200,4 +200,20 @@ class APP_EXPORT QgsMeshDatasetGroupTreeView : public QTreeView
     QgsMeshLayer *mMeshLayer = nullptr; // not owned
 };
 
+class QgsMeshDatasetGroupListModel: public QAbstractListModel
```

Comment:
is this class somewhere used?

Reply:
yes, in the mesh 3D symbol widget, to choose the dataset for rendering the Z component.

Reply:
ok